aexpect/remote.py

```diff
@@ -308,6 +312,15 @@ def handle_prompts(session, username, password, prompt=PROMPT_LINUX,
             elif match == 13:  # console prompt
                 logging.debug("Got console prompt, send return to show login")
                 session.sendline()
+            elif match == 14:  # VMware vCenter command prompt
+                # Some old vsphere version (e.x. 6.0.0) needs to enable first.
+                cmd = 'shell.set --enabled True'
+                logging.debug(
+                    "Got VMware VCenter prompt, send '%s' to enable shell first", cmd)
```

Comment:
I would break this into something more readable like
```python
logging.debug("Got VMware VCenter prompt, "
              "send '%s' to enable shell first", cmd)
```

Reply:
Yes, agree.